### PR TITLE
Add pycparser recipe

### PR DESCRIPTION
**Pull request by synapticarbors**

### CODE REVIEW

File: recipes/pycparser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - pycparser
+    - pycparser.ply
+
+about:
+  home: https://github.com/eliben/pycparser
+  license: BSD 3-clause
+  license_file: LICENSE
+  summary: 'C parser in Python'
+  decription: |
```
Comment (jakirkham) [inline recipes/pycparser/meta.yaml]: `decription` -> `description`

**Comment by synapticarbors** [comment]
@jakirkham I'm getting some strange sha256 checksum mismatches:

- The error or circleci: `SHA256 mismatch: '7959b4a74abdc27b312fed1c21e6caf9309ce0b29ea86b591fd2e99ecdf27f73' != 'fc51ccf9adbbb917a2ddcb386be852ae6dd36935e1e8b9a3e760201abfdbf77'`

- sha256 from link on https://pypi.org/project/pycparser/#files: 7959b4a74abdc27b312fed1c21e6caf9309ce0b29ea86b591fd2e99ecdf27f73

- sha256 from running `shasum -a 256 pycparser-2.14.tar.gz` on file downloaded from https://pypi.python.org/packages/6d/31/666614af3db0acf377876d48688c5d334b6e493b96d21aa7d332169bee50/pycparser-2.14.tar.gz: 7959b4a74abdc27b312fed1c21e6caf9309ce0b29ea86b591fd2e99ecdf27f73

- sha256 from running `shasum -a 256 pycparser-2.14.tar.gz` on file downloaded from 
https://pypi.io/packages/source/p/pycparser/pycparser-2.14.tar.gz: 
7959b4a74abdc27b312fed1c21e6caf9309ce0b29ea86b591fd2e99ecdf27f73

**Comment by synapticarbors** [comment]
@jakirkham nevermind, I think I made a cut and paste error and then was looking at the wrong recipe. Latest commit should have fixed things.

**Comment by jakirkham** [comment]
No worries. Happens. Thanks for letting us know.

**Comment by jakirkham** [comment]
LGTM

**Comment by jakirkham** [comment]
Any thoughts on this, @goanpeca?

**Comment by goanpeca** [comment]
:+1:

**Comment by jakirkham** [comment]
Thanks @synapticarbors.

**Pull request merged**